tests/unit/src/test/scala/tests/codeactions/CompanionObjectSuite.scala

```diff
@@ -0,0 +1,32 @@
+package tests.codeactions
+
+import scala.meta.internal.metals.codeactions.CreateCompanionObjectCodeAction
+
+class CompanionObjectSuite extends BaseCodeActionLspSuite("companionObject") {
+
+  check(
+    "insert-companion-object",
```

Comment:
I would also test some inner classes like:
```
class A {
  class B {
   <cursor here>
  }
}
```

Reply:
Sure! I would add a very comprehensive test for you!